MediaBrowser.Model/Syncplay/GroupUpdateType.cs

```diff
@@ -1,37 +1,53 @@
 namespace MediaBrowser.Model.Syncplay
 {
     /// <summary>
-    /// Enum GroupUpdateType
+    /// Enum GroupUpdateType.
     /// </summary>
     public enum GroupUpdateType
     {
         /// <summary>
         /// The user-joined update. Tells members of a group about a new user.
         /// </summary>
-        UserJoined = 0,
+        UserJoined,
         /// <summary>
         /// The user-left update. Tells members of a group that a user left.
         /// </summary>
-        UserLeft = 1,
+        UserLeft,
         /// <summary>
         /// The group-joined update. Tells a user that the group has been joined.
         /// </summary>
-        GroupJoined = 2,
+        GroupJoined,
         /// <summary>
         /// The group-left update. Tells a user that the group has been left.
         /// </summary>
-        GroupLeft = 3,
+        GroupLeft,
         /// <summary>
         /// The group-wait update. Tells members of the group that a user is buffering.
         /// </summary>
-        GroupWait = 4,
+        GroupWait,
         /// <summary>
         /// The prepare-session update. Tells a user to load some content.
         /// </summary>
-        PrepareSession = 5,
+        PrepareSession,
         /// <summary>
-        /// The not-in-group update. Tells a user that no group has been joined.
+        /// The not-in-group error. Tells a user that it doesn't belong to a group.
```

Comment:
```suggestion
        /// The not-in-group error. Tells a user that they don't belong to a group.
```
calling a user "it" doesn't sound polite 🤣

Reply:
How rude of me :laughing: